### DISCUSSION TITLE
замена переменных окружения $LIN на $LX**

### DISCUSSION THREAD

**Discussion by avmaisak** [Linux4Yourself/book#945] [Ideas]
- $LX
- $LX4
- $LXU

**Comment by Linuxoid85** [comment]
Не думаю, что это необходимо. Плюс LIN набирать действительно быстро, когда клавиши рябом друг с другом.

`LX` короткое, можно его, если решимся на замену. `LX4` - не знаю, зачем мешать цифры и буквы. `LXU` вообще непонятное название.
